cbopensource/connectors/taxii/cybox_parse.py

```diff
@@ -201,5 +254,27 @@ def cybox_parse_observable(observable, indicator, timestamp, score):
                                 'timestamp': timestamp,
                                 'link': link,
                                 'score': score})
+        if props.sha256:
+            iocs = {'sha256': []}
```

Comment:
This code looks like a good candidate to be refactored out into its own function.